### PR TITLE
Design database for tour training project

### PR DESCRIPTION
**Pull request by Phan-Hung-Mulodo**

### CODE REVIEW

**Comment by Phan-Hung-Mulodo** [comment]
@lam-vy-mulodo
done. plz review this pull request. thanks

File: fuel/app/migrations/001_create_areas.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	public function up()
+	{
+		\DBUtil::create_table('areas', array(
+			'id' => array('constraint' => 11, 'type' => 'int', 'auto_increment' => true, 'unsigned' => true),
+			'code' => array('type' => 'text'),
+			'name' => array('type' => 'text'),
+			'country' => array('type' => 'text'),
+			'country_name' => array('type' => 'text'),
+			'city' => array('type' => 'text'),
+			'city_name' => array('type' => 'text'),
```
Comment (si-lam-vy) [inline fuel/app/migrations/001_create_areas.php]: @Hung-Phan-Mulodo , pls replace code, name, country, country_name, city, city_name from text to varchar(length) type.

**Review by si-lam-vy — CHANGES_REQUESTED** [review]

File: fuel/app/migrations/002_create_tours.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+namespace Fuel\Migrations;
+
+class Create_tours
+{
+	public function up()
+	{
+		\DBUtil::create_table('tours', array(
+			'id' => array('constraint' => 11, 'type' => 'int', 'auto_increment' => true, 'unsigned' => true),
+			'code' => array('type' => 'text'),
```
Comment (si-lam-vy) [inline fuel/app/migrations/002_create_tours.php]: @Hung-Phan-Mulodo , pls replace code from text to varchar(length) type.

**Review by si-lam-vy — CHANGES_REQUESTED** [review]

File: fuel/app/migrations/002_create_tours.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+			'image' => array('type' => 'text'),
+			'title' => array('type' => 'text'),
+			'gross_min' => array('type' => 'text'),
+			'gross_max' => array('type' => 'text'),
```
Comment (si-lam-vy) [inline fuel/app/migrations/002_create_tours.php]: @Hung-Phan-Mulodo , pls replace gross_min, gross_max from text to int type.

**Review by si-lam-vy — CHANGES_REQUESTED** [review]

File: fuel/app/classes/model/area.php [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+		),
+		'Orm\Observer_UpdatedAt' => array(
+			'events' => array('before_update'),
+			'mysql_timestamp' => false,
+		),
+	);
+
+	protected static $_table_name = 'areas';
+
+}
```
Comment (si-lam-vy) [inline fuel/app/classes/model/area.php]: @Hung-Phan-Mulodo Hung
Pls use 4 spaces for indenting, not tabs. Ref https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md

**Review by si-lam-vy — CHANGES_REQUESTED** [review]

**Pull request closed without merging**